build/webpack.config.js

```diff
@@ -1,26 +1,73 @@
 const path = require("path");
 
-const themeEntries = require('./themes.js').themeEntries;
```

Comment:
do you have some formatter that changes the syntax?
please disable it for this project and use eslint configuration present in package.json

also do not commit your local changes here

Reply:
Thanks for the feedback @MV88 . Let me work on them.